lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -1281,11 +1310,13 @@ txHistoryFromEntity ti tip metas ins outs ws =
             { W.txInfoId =
                 getTxId txid
             , W.txInfoInputs =
-                map mkTxIn $ filter ((== txid) . txInputTxId . fst) ins
+                mkTxIn <$> filter ((== txid) . txInputTxId . fst) ins
```

Comment:
> Was this a hlint suggestion?

It _might_ have been 🤷🏻. (Though I can't reproduce it -- perhaps I implemented an `hlint` suggestion at some point and then made a later change which rendered the previous suggestion invalid...)

I must admit my natural instinct is to always use `fmap` (or `<$>`). Since we also use the `a <$> b` pattern quite a bit in the code, it seemed natural to write that.

Having said that, do you have a strong preference for `map` over `fmap` in this case? Is there a reason for preferring `map`? (Perhaps to make sure that it can't be used with some other `Functor`?)

Reply:
Either is fine by me, but if it doesn't need to be changed, there's no need to touch it.

Reply:
@rvl 

Fixed in d1e2ab04151ee25f907b64b9e174245ac0d895f7.